hal/halx86/smp/mpmemory.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * FILE:        hal/halx86/smp/mp.c
+ * PURPOSE:     Source file for private memory functions specific to SMP
+ * PROGRAMMERS:  Copyright 2021 Justin Miller <justinmiller100@gmail.com>
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include <hal.h>
+#include <smp.h>
+#include "smpp.h"
+#define NDEBUG
+#include <debug.h>
+
+/* GLOBALS *******************************************************************/
+
+/* AP spinup stub specific */
+#if 0
+extern PVOID APEntry;
+extern PVOID APEntryEnd;
+extern PVOID APSpinup;
+extern PVOID APSpinupEnd;
+#endif
```

Comment:
Please remove this if the variables are already declared in headers.

Reply:
I'm entirely why github is saying that these are disabled. But in the actual branch they're enabled. 
Never the less these are never declared in headers.

Reply:
oh, I did a per-commit review. Well, I guess it's fine, but I find it strange to write disabled code only to enable it in the next commit.

Reply:
I'm actually extremely terrible at git, so i probably did that on accident sorry about that!